src/components/inputMessage.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react'
+
+const InputMessage = ({ addMessage }) => {
+  const [message, setMessage] = useState('')
+
+  const handleMessageState = (e) => {
+    setMessage(e.target.value)
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    // Handle the submission of the message (e.g., send it to a server, update the state, etc.)
```

Comment:
It is better to check if input is not empty. Because if it is, message with no text will be sent, which is not good
```suggestion
    if (!message.length) {
      return
    }
    // Handle the submission of the message (e.g., send it to a server, update the state, etc.)
```